utils/helpers.js

```diff
@@ -181,3 +180,17 @@ export const getSample = (field) => {
     throw new Error(`Invalid model type:${field.type}`);
   }
 }
+
+export const calculateRefundTotal = (value, policy) => {
+  // if (!policy) {
+  //   return 0;
+  // }
+  // const { refundableUntil, refundablePercent } = policy;
```

Comment:
@closerearth this code snippet was suggested by Github Co-Pilot, I am leaving it here just as a proposal what `/cancellation-policy` response might include and how the refund calculation might be implemented. 

I cannot understand how to use the current response, please provide some explanation of what to do with `{default: 1,lastday: 0, lastmonth: 0.5, lastweek: 0}`.

I could also base the calculation purely on the description from the user journey, but I believe the endpoint has been created for a reason, so please advise on the best approach. 

I also think we'd better have a unit test for `calculateRefundTotal` because it is a crucial piece of the software, and it is better to test edge cases before they would happen in production. So if you are ok with that, after figuring out the implementation of the function, let's install [react testing library](https://testing-library.com/docs/react-testing-library/intro/), use some mocks and make a unit test

Reply:
IF (daysleftbefore booking.start > 30) {
   `refund ${policy.default * 100}%`
}
ELSE IF (daysleftbefore booking.start > 7) {
   `refund ${policy.lastmonth * 100}%`
}
ELSE IF (daysleftbefore booking.start > 1) {
   `refund ${policy.lastweek * 100}%`
}
ELSE IF (daysleftbefore booking.start > 0) {
   `refund ${policy.lastday * 100}%`
}
ELSE    {
    'This booking is in the past'. (error state - cannot be cancelled)
}